src/collections/Menu/MenuItem.js

```diff
@@ -8,16 +8,21 @@ import {
 } from '../../lib'
 
 function MenuItem(props) {
-  const { active, children, className, header } = props
+  const { active, children, className, header, href, link, onClick } = props
   const classes = cx(
     className,
     useKeyOnly(active, 'active'),
     useKeyOnly(header, 'header'),
+    useKeyOnly(link, 'link'),
     'item',
   )
+  const handleClick = (e) => {
+    if (onClick) onClick(e)
+  }
   const rest = getUnhandledProps(MenuItem, props)
+  const ItemComponent = href || onClick ? 'a' : 'div'
 
-  return <a {...rest} className={classes}>{children}</a>
+  return <ItemComponent {...rest} className={classes} onClick={handleClick}>{children}</ItemComponent>
```

Comment:
> I'm thinking of doing this today. The getElementType() method would hold the logic for returning the value of the as prop, else if props.link return a, else div.

Seems, it's not good idea, prop `link` might be passes to `class`, [this on `Step`](https://github.com/TechnologyAdvice/stardust/blob/20e9871b59dff33fa9149736d4b903683fa1ccfe/src/elements/Step/Step.js#L27).

![image](https://cloud.githubusercontent.com/assets/14183168/17749057/e9c22d7e-64c4-11e6-8d1d-e1044f46ff6b.png)